fiftyone_devicedetection_cloud/tests/test_properties.py

```diff
@@ -114,7 +114,10 @@ def test_value_types(self):
             expected_type = propertymeta["type"]
             if(property in dd_property_dict and property not in exclude_properties):
                 dd_property_value = dd_property_dict[property]
-                value = dd_property_value.value()
-                self.assertIsNotNone("Property: " + property +" is not present in the results.", dd_property_value)
-                self.assertTrue("Expected type for " + property + " is " + expected_type + 
-                " but actual type is " + get_value_type(value), is_same_type(value, expected_type))
+                try:
+                    value = dd_property_value.value()
+                    self.assertIsNotNone("Property: " + property + " is not present in the results.", dd_property_value)
```

Comment:
you need to reverse the order of arguments in the `assertIsNotNone` call.  it should be `(testedValue, message)`.